CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 - A `WorkerSystem` now exposes the underlying Worker's `IsConnected` property. [#1217](https://github.com/spatialos/gdk-for-unity/pull/1217)
 
+### Fixed
+
+- The Deployment Launcher window no longer saves changes after every input. [#1219](https://github.com/spatialos/gdk-for-unity/pull/1219)
```

Comment:
Its not clear why this is under "Fixed" from this description. Why does a user care about this?

Reply:
I was debating whether to put this in changed or fixed

should I:
a - move to changed
b - change this to something like "Fixed a performance issue where the Deployment Launcher window would feel slow because xyz.."